src/core/components/profile/stepCard.component.tsx

```diff
@@ -0,0 +1,183 @@
+import { Button, makeStyles, Card, Typography, Box } from "@material-ui/core"
+import React from "react"
+import { Link } from "react-router-dom"
+import stepCardConstant from "../../constants/stepCard.constant"
+
+import { useApplicationStateContext } from "../../providers/applicationState.provider"
+
+const useStyles = makeStyles(theme => ({
+  card: {
+    padding: theme.spacing(4),
+    boxShadow: "0px 100px 257px rgba(0, 0, 0, 0.07), 0px 18.576px 34.4894px rgba(0, 0, 0, 0.0269069)",
+    borderRadius: "10px",
+    margin: theme.spacing(2.25)
+  },
+  stepNo: {
+    borderRadius: "100%",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+
+    width: 56,
+    height: 56,
+    margin: theme.spacing(1)
+  },
+  redCircle: {
+    background: theme.palette.primary.main,
+    color: "#FFFFFF" //theme.palette.gray[0]
+  },
+  outlinedRedCircle: {
+    border: `3px solid ${theme.palette.primary.main}`,
+    color: theme.palette.primary.main //theme.palette.gray[0]
+  },
+  outlinedGrayCircle: {
+    border: "3px solid #BFBFBF", //theme.palette.gray[200]
+    color: "#BFBFBF" //theme.palette.gray[200]
+  },
+  stepIndicator: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    width: "150px",
+    marginRight: theme.spacing(5),
+    [theme.breakpoints.down("sm")]: {
+      marginRight: theme.spacing(0),
+      marginBottom: theme.spacing(2)
+    }
+  },
+  text: {
+    textAlign: "left",
+    maxWidth: "520px"
+  },
+  title: {
+    fontWeight: 500,
+    textTransform: "uppercase",
+    [theme.breakpoints.down("sm")]: {
+      textAlign: "center",
+      marginBottom: theme.spacing(2)
+    }
+  },
+  blackText: {
+    color: "#262626" //theme.palette.gray[800]
+  },
+  redText: {
+    color: theme.palette.primary.main
+  },
+  grayText: {
+    color: "#BFBFBF" //theme.palette.gray[200]
+  },
+  content: {
+    color: "#8C8C8C" //theme.palette.gray[300]
+  },
+  button: {
+    marginTop: theme.spacing(2.5),
+    padding: theme.spacing(0.5, 1.5),
+    borderRadius: "10px",
+    flexGrow: 1
+  },
+  solid: {
+    color: "white",
+    background: theme.palette.primary.main,
+    "&:hover": {
+      background: theme.palette.primary.dark
+    }
+  },
+  outlined: {
+    color: theme.palette.primary.main,
+    borderColor: theme.palette.primary.main
+  },
+  buttonContainer: {
+    display: "flex",
+    columnGap: theme.spacing(2),
+    flexWrap: "wrap",
+    maxWidth: "520px"
+  },
+  flexRow: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "left",
+    flexGrow: 1,
+    [theme.breakpoints.down("sm")]: {
+      flexWrap: "wrap",
+      justifyContent: "center"
+    }
+  },
+  flexCol: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "left"
+  }
+}))
+
+export interface StepCardProps {
+  step: 1 | 2 | 3 | 4 | 5 | 6
+  status: "complete" | "inProgress" | "incomplete"
+  pass: "pass" | "fail"
```

Comment:
Didn't use boolean because
![image](https://user-images.githubusercontent.com/57129145/140740733-7e8c4d41-4d52-4076-ba38-a3161c1315fc.png)

But maybe I could use `"true"/"false"` string

Reply:
weird